packaging/tags.py

```diff
@@ -101,6 +105,16 @@ def parse_tag(tag):
     return frozenset(tags)
 
 
+def _get_config_var(name, warn=False):
```

Comment:
So with this being false, how will this ever be true such that logging actually occurs? Is the assumption people will manually change locally when they need to? Or is it in case logging comes in the future for the whole the project or through some API?

Reply:
Right, great point. I think what would be best is that users can turn on logging via an argument to `sys_tags` and this is propagated through. I feel like this would be easiest to integrate into `pip` and would have the least effect on other `packaging.tags` users. I added a commit which does this.